server/start.go

```diff
@@ -81,7 +82,7 @@ which accepts a path for the resulting pprof file.
 	cmd.Flags().String(flagTraceStore, "", "Enable KVStore tracing to an output file")
 	cmd.Flags().String(flagPruning, "syncable", "Pruning strategy: syncable, nothing, everything")
 	cmd.Flags().String(
-		FlagMinGasPrices, "",
+		FlagMinGasPrices, "0.00000001"+types.DefaultBondDenom,
```

Comment:
It is best not to modify it here, otherwise the config file will not take effect

Reply:
ok. Remove it now.